go/cmd/vttablet/vttablet.go

```diff
@@ -74,7 +74,8 @@ func main() {
 	}
 
 	// creates and registers the query service
-	qsc := tabletserver.NewServer()
+	ts := topo.Open()
```

Comment:
Is this function idempotent, or can it be made so? If so, we can have TabletServer start this as a sub-service internally, which will be in line with the rest of the sub-services. We'll also reduce one dependency.